src/main/org/apache/tools/ant/taskdefs/optional/ssh/ScpToMessage.java

```diff
@@ -36,7 +36,7 @@
 public class ScpToMessage extends AbstractSshMessage {
 
     private static final int HUNDRED_KILOBYTES = 102400;
-    private static final int BUFFER_SIZE = 1024;
+    private static final int BUFFER_SIZE = 100*1024;
```

Comment:
Any reason you don't make BUFFER = HUNDRED_KILOBYTES, since that constant is defined just above?